installer/client/src/pages/WelcomeScreen.vue

```diff
@@ -108,14 +108,16 @@ export default {
   },
   methods: {
     toggleModal() {
+      if (this.selected === 'install') {
+        navigate('/installer/database-config');
```

Comment:
why this? we do this in closeModal function right? which also check if user has accepted it

Reply:
we don't need to show warning pop up modal when we have a fresh installation